### DISCUSSION TITLE
New category course - problem

### DISCUSSION THREAD

**Discussion by KMaN1985** [chamilo/chamilo-lms#5830] [General]
When adding a new course category, use the "add category" button. The browser window turns white (URL address: **/main/admin/course_category.php?action=add&category=&id=_** ), there is no new category added in the platform. How can I solve the problem?

**Comment by ywarnier** [comment]
Hi @KMaN1985 
You probably would benefit from this patch https://github.com/chamilo/chamilo-lms/commit/48f192df355262034ef186870b1d53a47e5dfdc5 
Let us know if you need more help understanding and let us know if that works for you. This issue was initially reported here https://github.com/chamilo/chamilo-lms/issues/5592 but we are very late publishing 1.11.28 which includes it.

**Reply by KMaN1985** [reply]
Thank you, I modified it according to your instructions. And It works without any problems. Thank you again for your help.
